.github/workflows/snapshot-publish.yml

```diff
@@ -40,6 +40,7 @@ jobs:
       - name: Scan verification repo
         working-directory: src/Microsoft.ComponentDetection
         run: dotnet run scan --Verbosity Verbose --SourceDirectory ${{ github.workspace }}/test/Microsoft.ComponentDetection.VerificationTests/resources --Output ${{ github.workspace }}/output
+                                                 --DockerImagesToScan "docker.io/library/debian@sha256:9b0e3056b8cd8630271825665a0613cc27829d6a24906dc0122b3b4834312f7d,mcr.microsoft.com/cbl-mariner/base/core:2.0,alpine@sha256:1304f174557314a7ed9eddb4eab12fed12cb0cd9809e4c28f29af86979a3c870"
```

Comment:
Please use a digest for the Mariner image, and use a fully qualified reference for the alpine image `docker.io/library/alpine@sha256:...`